Add quickstarts for the catalog connectors (#133)
spiceai/quickstarts: catalogs/unity_catalog/README.md

# Unity Catalog Connector

> The Unity Catalog Connector allows you to connect to your tables in a Unity Catalog and query them using Spice.

Reviewer: ```suggestion
The Unity Catalog Connector makes querying tables in a Unity Catalog with Spice simple.
```

> Note: This is for the [open-source version of Unity Catalog](https://www.unitycatalog.io/). For connecting to the Databricks Unity Catalog Connector, see the [Databricks Unity Catalog Connector](../databricks/README.md).

Reviewer: ```suggestion
Note: This quickstart applies to the [open-source version of Unity Catalog](https://www.unitycatalog.io/). To get started with the Databricks Unity Catalog Connector, see the [Databricks Unity Catalog Connector quickstart](../databricks/README.md).
```

    from: uc
```

The Unity Catalog connector only supports Delta Lake tables and requires specifying the object store credentials to connect to the Delta Lake tables.

Visit the documentation for more information configuring the [Unity Catalog Connector](https://docs.spiceai.org/components/catalogs/unity-catalog).

## Step 3. Configure the object store credentials

Using the Spice CLI, set the credentials needed to connect to the Delta Lake tables provided by the Unity Catalog Connector.

### Using Delta Lake directly against AWS S3
`spice login delta_lake --aws-region <aws-region> --aws-access-key-id <aws-access-key-id> --aws-secret-access-key <aws-secret-access-key>`

### Using Delta Lake directly against Azure Blob Storage
`spice login delta_lake --azure-storage-account-name <account-name> --azure-storage-access-key <access-key>`

### Using Delta Lake directly against Google Cloud Storage
`spice login delta_lake --google-service-account-path /path/to/service-account.json`

## Step 5. Start the Spice runtime

```bash
spice run
```

## Step 6. Query a dataset

```bash
spice sql
```

```sql

> SELECT * FROM uc.<SCHEMA_NAME>.<TABLE_NAME> LIMIT 10;

Reviewer: Example output.

Author: Added
